From: Johan Van de Wauw <johan.vandewauw@gmail.com>
Date: Fri, 12 Jun 2015 19:28:19 +0200
Subject: Rename rio to rasterio (Closes: #788463)

There is already another package providing a binary "rio".

--- a/setup.cfg
+++ b/setup.cfg
@@ -26,7 +26,7 @@ group = root
 
 [options.entry_points]
 console_scripts =
-    rio = rasterio.rio.main:main_group
+    rasterio = rasterio.rio.main:main_group
 rasterio.rio_commands =
     blocks = rasterio.rio.blocks:blocks
     bounds = rasterio.rio.bounds:bounds
